src/main/java/Duke.java

```diff
@@ -35,6 +37,26 @@ private static void readAndRespond() {
         }
     }
 
+    private static boolean isValidMark(String input, String[] words) {
+        return input.contains("mark") && words.length == 2 && isInt(words[1]);
```

Comment:
Coding Quality: can put the return in one variable first.
```suggestion
isValidString = input.contains("mark") && words.length == 2 && isInt(words[1]);
return isValidString;
```

Reply:
The same goes for line 44, 48, 53, 57

Reply:
☝️ I agree with steven's comment! Perhaps putting the return result in an aptly named variable like `isValidTask` could make this portion of the code easier to understand. However, since the method name already gives a good idea of what the return result is supposed to be, this suggestion may be unnecessary.